projects/angular-editor/src/lib/angular-editor.component.ts

```diff
@@ -34,8 +34,7 @@ import {isDefined} from './utils';
       provide: NG_VALUE_ACCESSOR,
       useExisting: forwardRef(() => AngularEditorComponent),
       multi: true
-    },
-    AngularEditorService
+    }
```

Comment:
I think we not need to remove this line.  Only in ToolbarComponent.

Reply:
Well as AngularEditorService defines:

`@Injectable({
  providedIn: 'root'
})`

It is not useful to keep it in the component. Or maybe we should remove the providedIn: 'root' which could be cleaner for a lib.

Reply:
Actually especially if a single instance of the service is required per component it should not be defined neither in the module neither in root actually and you should have the provided back...